app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -0,0 +1,133 @@
+module Api
+  module V2
```

Comment:
I started with routes but then I realized that a lot of parameters would change, like `params[:config_templates]` vs `params[:provisioning_templates]`, also I'd have to change rabl templates to render response based on how the user asked... Also this way when we drop the config_templates API in V3 we can just not include the whole controller and views instead of modifying it's content (so it will be trivial task).